src/main/java/com/logbook/backend/logbookbe/domain/user/controller/UserController.java

```diff
@@ -1,38 +1,62 @@
 package com.logbook.backend.logbookbe.domain.user.controller;
 
+import com.logbook.backend.logbookbe.domain.user.controller.dto.LoginRequest;
+import com.logbook.backend.logbookbe.domain.user.controller.dto.SignupRequest;
+import com.logbook.backend.logbookbe.domain.user.usecase.RefreshToken;
+import com.logbook.backend.logbookbe.domain.user.usecase.UserSignup;
+import com.logbook.backend.logbookbe.global.jwt.dto.JwtResponse;
 import com.logbook.backend.logbookbe.domain.user.model.User;
 import com.logbook.backend.logbookbe.domain.user.service.UserService;
+import com.logbook.backend.logbookbe.domain.user.usecase.UserLogin;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.annotations.Parameter;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletResponse;
 
 @RestController
 @RequestMapping("/api")
```

Comment:
```suggestion
@RequestMapping("/api/user")
```
로 엔드포인트를 명확하게 했으면 좋겠어요~!!